### ISSUE TITLE
Doesn't work on 2022.9 (beta)

### ISSUE THREAD

**Issue by TheFes** [leikoilja/ha-google-home#602]
## Describe the bug

`Setup failed for custom integration google_home: Requirements for google_home not found: ['glocaltokens==0.6.7'].`

## Version of the integration

If you are unsure about the version check the [github release here](https://github.com/leikoilja/ha-google-home/releases).

```text
v1.9.14

```

## Logs

```log
2022-08-31 21:48:51.856 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration google_home: Requirements for google_home not found: ['glocaltokens==0.6.7'].

homeassistant.requirements.RequirementsNotFound: Requirements for google_home not found: ['glocaltokens==0.6.7'].
```

```text
```

**Comment by pjponymoore** [comment]
Same Issue with 2022.9 (beta)

Logger: homeassistant.setup
Source: setup.py:184
First occurred: 22:18:56 (1 occurrences)
Last logged: 22:18:56

Setup failed for custom integration google_home: Requirements for google_home not found: ['glocaltokens==0.6.7'].

**Comment by tjorim** [comment]
Not sure what the bug is. It clearly fails on requirements, but AFAIK glocaltokens has no dependencies and no other HA core integration uses it either. I run my HA OS in a VM, SSH'd into the homeassistant container and did a pip install. No obvious errors (was on my mobile so could not scroll up tho).
![Screenshot_20220902-092430.png](https://user-images.githubusercontent.com/1294876/188125942-fe2002fa-3dcc-4ead-88b2-ef0ddc0e8869.png)

**Comment by KapJI** [comment]
Should be fixed in 1.9.15.

**Comment by pjponymoore** [comment]
Sorted with update

Thanks for time on this

**Comment by TheFes** [comment]
Thanks, indeed solved in 1.9.15
